src/main/java/org/schedulesdirect/api/Station.java

```diff
@@ -80,13 +80,13 @@ public void writeImageToFile(File dest) throws IOException {
 		/**
 		 * @return the url
 		 */
-		URL getUrl() {
+		public URL getUrl() {
```

Comment:
Is there a reason to make these two public?  Don't think clients should need to access nor modify this internal state.